### ISSUE TITLE
bug: default configuration value of jsx-boolean-value

### ISSUE THREAD

**Issue by remitbri** [yannickcr/eslint-plugin-react#210]
The [documentation](../blob/master/docs/rules/jsx-boolean-value.md) for this rule says 
> The default value of this option is "never"

But ESLint using `"react/jsx-boolean-value": 2` doesn't flag an error with
```jsx
<Foo bar={true} />
```

I think the problem comes from
```js
  var configuration = context.options[0] || {};
```
instead of 
```js
  var configuration = context.options[0] || 'never';
```
in the [rule file](../blob/master/lib/rules/jsx-boolean-value.js)

**Comment by yannickcr** [comment]
Good catch! I'll fix this.
